packages/lambda-tiler/src/routes/tile.style.json.ts

```diff
@@ -112,6 +113,13 @@ export async function styleJsonGet(req: LambdaHttpRequest<StyleGet>): Promise<La
     if (tileSet == null) return NotFound();
     if (tileSet.type !== TileSetType.Raster) return NotFound();
     return tileSetToStyle(req, tileSet, apiKey);
+  } else {
+    const layers = styleConfig.style.layers;
+    styleConfig.style.layers = [];
+    for (const layer of layers) {
+      if (excludeLayers.find((l) => l.toLowerCase() === layer.id.toLowerCase())) continue;
```

Comment:
this is not very efficient, granted the layer array is quite small..

you could however do something like

```javascript
 const excluded =  new Set(excludeLayers)

for (const layer of layers) { 
  if (excluded.has(l.id)) continue; 
  ....
}
```